src/test/java/com/yodle/vantage/functional/VersionCreateAndUpdateTest.java

```diff
@@ -148,7 +148,7 @@ public void updateVersionWithoutDepsToHaveDeps() {
     @Test
     public void updateVersionWithDepsToHaveMoreDeps() {
         Dependency resolvedDependency = createDependency("dependency component", "dependencyversion");
-        Version v = createVersion("component", "version", new HashSet<>(), Sets.newHashSet());
+        Version v = createVersion("component", "version", new HashSet<>(), Sets.newHashSet(resolvedDependency));
```

Comment:
I actually had a test that tested this very scenario...but i forgot to set the dependency on the first update.  :(